### DISCUSSION TITLE
I'm having issues setting up VisualStudioCode on my Mac

### DISCUSSION THREAD

**Discussion by creativitycrackhead** [microsoft/vscode-discussions#565] [Extension Development QnA]
I installed the C++ extension. But now running basic code doesn't show up in the output or in the terminal. Here's the code:

--------------------------------------------------------------------------------------------------------------
#include <iostream>
#include <string>

using namespace std;

int main()
{
    cout<<"hello world"<<endl;
}
----------------------------------------------------------------------------------------

Under output, I get this: 'IntelliSense process crash detected.'
If I'm not using it correctly, please let me know. I'm a visual studio vet on windows, but I just CANNOT get it running as expected on my Mac. I really need help here...

**Comment by olguzzar** [comment]
Hi there, this is a forum specifically for those creating their own extensions. I suggest going through the [C++ for VS Code docs](https://code.visualstudio.com/docs/languages/cpp), and if you're still having issues after troubleshooting, you can file a bug on the main [VS Code repo](https://github.com/microsoft/vscode/issues). Thanks!
